pkg/kubelet/apis/config/BUILD

```diff
@@ -38,6 +38,7 @@ filegroup(
         ":package-srcs",
```

Comment:
change `pkg/kubelet/apis/config/OWNERS` to:

```
# See the OWNERS docs at https://go.k8s.io/owners

# Disable inheritance as this is an api owners file
options:
  no_parent_owners: true
approvers:
- api-approvers
reviewers:
- sig-node-api-reviewers
```

Reply:
done